renovate.json

```diff
@@ -15,8 +15,7 @@
         "twig/twig",
         "vlucas/phpdotenv"
       ],
-      "matchUpdateTypes": ["minor", "patch"],
-      "enabled": false
+      "addLabels": ["test only - do not merge"]
```

Comment:
This is great! If we ever have trouble with PRs getting merged by mistake, we could probably create an auto-request-changes action too (as Renovate doesn't have this as an option).